pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/assignment/segment/OfflineSegmentAssignment.java

```diff
@@ -44,6 +40,13 @@
  */
 public class OfflineSegmentAssignment extends BaseSegmentAssignment {
```

Comment:
(nit), while it may not make sense for `RealtimeSegmentAssignment`, for `OfflineSegmentAssignment`, we can consider making `InstancePartitionsType.OFFLINE` an instance variable as it is referenced 9 times in this class.

Reply:
InstanceVariable as in? The comparison would be still same correct?